apps/server/src/index.ts

```diff
@@ -75,6 +76,7 @@ app.use(
   })
 );
 
+app.useGlobalFilters(new ExceptionLogger());
```

Comment:
ExceptionLogger will throw an error in the previous code. I change to another style and it works

@thorseraq